pkgs/development/python-modules/ducc0/default.nix

```
@@ -2,7 +2,7 @@
 
 buildPythonPackage rec {
   pname = "ducc0";
-  version = "0.33.0";
+  version = "0.34.0";
   format = "setuptools";
```

```suggestion
  pyproject = true;
```
Could you replace this old parameter to avoid legacy build?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters

---

Thank you for your comment! I implemented the change as proposed and made the overall formatting compatible with `nixfmt-rfc-style`.